### DISCUSSION TITLE
Alert when the Wi-Fi signal gets low or is unreliable

### DISCUSSION THREAD

**Discussion by p808u2** [ratgdo/homekit-ratgdo#176] [Ideas]
It may be interesting to know (HomeKit notification - is that a thing?) once the wifi signal becomes low or unreliable. Perhaps triggering a reboot of the ratgdo to reattempt a new connection to the AP?

The goal being to know why sometimes the garage door is "not responsive".

**Comment by dkerr64** [comment]
I'm not sure how you could do this through HomeKit, but you can monitor the signal strength at the web UI and also from a command line with `curl http://<ip-address>/status.json`

I don't think rebooting would do much good.  But again you can trigger that from command line too with `curl -s -X POST http://<ip-address>/reboot` so you could write your own script to monitor things.
